src/docbook-tool.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+(function () {
+    /**
+     * @param {NodeListOf<HTMLElement>} unselectedListElements
+     * @param {HTMLElement} selectedListElement
+     * @param {boolean} applySelectedClass
+     * @param {boolean} applyHiddenClass
+     */
+    function selectFromList(unselectedListElements, selectedListElement, applySelectedClass, applyHiddenClass)
+    {
+        for (let i = 0; i < unselectedListElements.length; i++) {
+            if (applySelectedClass && unselectedListElements[i].classList.contains('selected')) {
+                unselectedListElements[i].classList.remove('selected');
+            }
+            if (applyHiddenClass && ! unselectedListElements[i].classList.contains('hidden')) {
+                unselectedListElements[i].classList.add('hidden');
+            }
+        }
+
+        if (applySelectedClass) {
+            selectedListElement.classList.add('selected');
+        }
+        if (applyHiddenClass) {
+            selectedListElement.classList.remove('hidden');
+        }
+    }
+
+    function fragmentRoute(pathWithHash) {
+        const path = pathWithHash.substring(1);
+
+        const pathParts = path.split("/");
+
+        let pageTitle = 'WISP Reporting Tool Docs';
```

Comment:
This is the page title I mentioned that needs to change depending on what is using it :thinking: